### ISSUE TITLE
[Candidate 4v4 Twisted Treeline ZH v1] A few soldiers move down the ramp after evacuating towers

### ISSUE THREAD

**Issue by xezon** [TheSuperHackers/GeneralsRankedMaps#785]
## Observed behavior

A few soldiers move down the ramp after evacuating towers.

## Expected behavior

No soldiers move down the ramp after evacuating towers.

### 2 of 10 soldiers walk down the ramp after evacuation

![shot_20240518_124647_37](https://github.com/TheSuperHackers/GeneralsRankedMaps/assets/4720891/e3841aee-9bd3-41f1-ad84-000a0939fdee)

### 3 of 10 soldiers walk down the ramp after evacuation

![shot_20240518_124658_38](https://github.com/TheSuperHackers/GeneralsRankedMaps/assets/4720891/a4727421-f647-476e-910e-ff6252df3070)

**Comment by NoMerciBKK** [comment]
units evac in a circle based arround the building, if there is impassible terrain they evac in a bigger circle, that happens on both sides, so as designed

**Comment by xezon** [comment]
On most maps this does not happen because bunkers are rarely placed this way.

**Comment by NoMerciBKK** [comment]
![image](https://github.com/TheSuperHackers/GeneralsRankedMaps/assets/146106317/fce2f9c2-2f88-442d-b9b9-86934f52fb5b)
![image](https://github.com/TheSuperHackers/GeneralsRankedMaps/assets/146106317/9183f814-70e9-467c-a322-8488e414a0c0)


if you put a cricle arround there you know why they are running down the ramp

**Comment by xezon** [comment]
Note this issue also happened with
* #684
and that was fixed.

**Comment by NoMerciBKK** [comment]
yes but there you have 180° not impassible terrain here you have a lot more impassible terrain

**Comment by NoMerciBKK** [comment]
> Note this issue also happened with
> 
> * [[Candidate 4v4 Umbrella Corps ZH v1] One soldier will move down after evacuating tower #684](https://github.com/TheSuperHackers/GeneralsRankedMaps/issues/684)
> 
> and that was fixed.

there we had an similar problem that the building was to close to the edge
